src/spartan_tcp_handler.erl

```diff
@@ -39,7 +39,6 @@ callback_mode() ->
     state_functions.
 
 init([Ref, Socket, Transport, Opts]) ->
-    {ok, _} = timer:kill_after(?TIMEOUT * 5),
```

Comment:
should we turn this into a timeout info message? or is it already covered by the timeout in `uninitialized/1`?